src/components/index.ts

```diff
@@ -17,10 +17,13 @@ export * from "./table";
 export * from "./xDragBoxLayer";
 export * from "./yDragBoxLayer";
 
-export class Alignment {
-  static TOP = "top";
-  static BOTTOM = "bottom";
-  static LEFT = "left";
-  static RIGHT = "right";
-  static CENTER = "center";
-}
+/**
+ * @deprecated just use string literals for alignment
```

Comment:
why? constants are much easier to work with, and you get autocomplete.

the big win here IMO is that string literals are _supported_ if that's what you wanna do, but you can still use the helpful constants.